app/admin/question_answers.rb

```diff
@@ -23,6 +23,42 @@
     actions
   end
 
+  csv do
+    column :id
+    column(:user_id) do |question_answer|
+      question_answer.user_id
+    end
+    column(:user_first_name) do |question_answer|
+      question_answer.user.first_name
+    end
+    column(:user_middle_name) do |question_answer|
+      question_answer.user.middle_name
+    end
+    column(:user_family_name) do |question_answer|
+      question_answer.user.family_name
+    end
+    column(:consent_question) do |question_answer|
+      question_answer.consent_question.question
+    end
+    column :answer
+    column(:answer_redcap_field) do |question_answer|
+      redcap_response = UploadRedcapDetails.question_answer_to_redcap_response(
```

Comment:
All changes to other files in this PR were to provide this `UploadRedcapDetails.question_answer_to_redcap_response` function. `app/lib/upload_redcap_details.rb` (formerly `app/jobs/upload_redcap_details_job.rb`) is where this function was added and contains lots of refactoring. Changes to the tests and `app/models/question_answer.rb` are to accommodate the refactoring.